### PR TITLE
Improve experiment deletion UX

### PR DESCRIPTION
**Pull request by andrewmchen**
This PR improves the experiment deletion UX in two ways.
- The root route always shows the experiment view with ID 0. We should instead show the active experiment with the smallest ID.
- When there are no active experiments we should should an "no experiments found" view.

Screenshot
screen shot 2018-10-09 at 1 52 54 pm

Design wise, the `HomePage` will issue a request to list experiments and the `ExperimentPage` will no longer care about the route. (It should always be passed an experiment ID explicitly.)

### CODE REVIEW

File: mlflow/server/js/src/components/HomeView.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const experiments = [
+  Fixtures.createExperiment({ experiment_id: '1', name: '1', lifecycle_stage: 'deleted'}),
+  Fixtures.createExperiment({ experiment_id: '3', name: '3', lifecycle_stage: 'active'}),
+  Fixtures.createExperiment({ experiment_id: '2', name: '2', lifecycle_stage: 'active'}),
+];
+
+test('getFirstActiveExperiment works', () => {
+  expect(getFirstActiveExperiment(experiments).experiment_id).toEqual('2');
+});
```
Comment (smurching) [inline mlflow/server/js/src/components/HomeView.test.js]: Micronit: newline at EOF

File: mlflow/server/js/src/components/NoExperimentView.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import Colors from '../styles/Colors';
+import overflow from '../static/project.svg';
+
+export default class NoExperimentView extends Component {
+  render() {
+    return (
+      <div>
+        <img
+          className='center'
+          alt="404 Not Found"
```
Comment (smurching) [inline mlflow/server/js/src/components/NoExperimentView.js]: It looks like `alt` is also used for accessibility purposes (i.e. screen readers will read out the text in the alt tag): https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Attributes.

Maybe instead we could put a description here like "No experiments exist"? Also, the icon for some reason doesn't appear as it does in your screenshot for me (i.e. the colors & stuff are missing). Not a big deal but just wanted to make sure you knew:

screen shot 2018-10-09 at 2 26 55 pm

**Review by smurching — COMMENTED** [review]
Looks pretty good, just one small comment + nit!

File: mlflow/server/js/src/components/NoExperimentView.js (hunk 2 above)
Comment (andrewmchen) [reply mlflow/server/js/src/components/NoExperimentView.js]: Forgot to add the new image! It's there now.

**Review by smurching — APPROVED** [review]
LGTM thanks for the update!

**Pull request merged**